pipeline/pipeline_tasks/api_calls/congressional_districts.py

```diff
@@ -1 +1,56 @@
 """Hit Google Civic API to get congressional districts"""
+
+import os
+
+import requests
+
+import re
+
+import pandas as pd
+
+def get_congressional_dist_by_addr(df):
+    """
+    Get Contressional District from Google Maps
+
+    Keyword Args:
+      df: DataFrame with business information to request congressional districts
+    """
+
+    url = 'https://www.googleapis.com/civicinfo/v2/representatives'
+
+    if df is None or df.empty:
+        print("Incoming dataframe is empty.")
+        return
+
+    df['cong_dist'] = None
```

Comment:
Let's rename this to `congressional_district_google_civic`? and then we can compare to `congressional_district`

Reply:
Done and pushed.